resolver/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+        <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+                <file>test-run.log</file>
+                <append>false</append>
+                <!-- set immediateFlush to false for much higher logging throughput -->
+                <immediateFlush>false</immediateFlush>
+                <!-- encoders are assigned the type
+                     ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+                <encoder>
+                        <pattern>bloop %d{HH:mm:ss.SSS} [%-5level] %logger{35} - %msg%n%rEx</pattern>
+                </encoder>
+        </appender>
+
+
+        <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+                <encoder>
+                        <pattern>bloop %d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
```

Comment:
bloop?

Reply:
Copy pasted from james-project/testing-base. No idea wha&t it means, I trust @mbaechler .

Reply:
it's clearly a debug statement